AsyncDisplayKit/ASMapNode.mm

```diff
@@ -159,6 +161,15 @@ - (void)setRegion:(MKCoordinateRegion)region
 
 - (void)takeSnapshot
 {
+  // set _snapshotNeeded to YES so if layout changes in the future, we'll try snapshotting again.
```

Comment:
Update ivar name in comment; it should also mention something like "if we are zero-sized, ..."